Objects/frameobject.c

```diff
@@ -1802,11 +1802,25 @@ PyDoc_STRVAR(clear__doc__,
 static PyObject *
 frame_sizeof(PyFrameObject *f, PyObject *Py_UNUSED(ignored))
 {
-    Py_ssize_t res;
-    res = offsetof(PyFrameObject, _f_frame_data) + offsetof(_PyInterpreterFrame, localsplus);
+    Py_ssize_t base = offsetof(PyFrameObject, _f_frame_data)
```

Comment:
I think there is no need to change `frame_sizeof` as `_PyCode_Validate` ensures we can't overflow.